CRM/Utils/Check/Component/Env.php

```diff
@@ -116,6 +116,27 @@ public function checkPhpMysqli() {
   }
 
   /**
+   * @return array
+   */
+  public function checkPhpEcrypt() {
+    $messages = array();
+    $test_pass = 'iAmARandomString';
+    $encrypted_test_pass = CRM_Utils_Crypt::encrypt($test_pass) 
```

Comment:
@adixon there is whitespace at the end of this line preventing Jenkins doing its thing

Reply:
Worse than that, there's a missing semi-colon. Just fixed.